point-focus.js

```diff
@@ -1,7 +1,7 @@
 document.addEventListener("DOMContentLoaded", function () {
     document.body.addEventListener("mousedown", function (evt) {
-        if (!evt.target.setSelectionRange || evt.target.role === 'textbox' || evt.target.hasAttribute("disable-point-focus")) {
-                evt.target.setAttribute("point-focused", true); 
+        if (!t.setSelectionRange && t.getAttribute("role") !== 'textbox' || t.hasAttribute("disable-point-focus")) {
```

Comment:
Add some brackets around the &&-ed terms so we don't need to remember operator precedence? :smile: